### PR TITLE
Add loading state

### PR DESCRIPTION
**Pull request by nadinho**

### CODE REVIEW

File: src/components/Loading.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { keyframes } from '@emotion/core';
+
+const rotate360 = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
```
Comment (marwinburesch) [inline src/components/Loading.js]: ANIMATIONS! 🙃

File: src/components/Loading.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const Spinner = styled.div`
+  animation: ${rotate360} 1s linear infinite;
+  transform: translateZ(0);
```
Comment (marwinburesch) [inline src/components/Loading.js]: I don't think you need this

File: src/pages/Add.js [hunk 3]
```diff
@@ -23,6 +28,19 @@ export default function Add() {
     };
 
     await postFormular(formular);
+    history.push('/Thanks');
+  }
+  const Div = styled.div`
```
Comment (marwinburesch) [inline src/pages/Add.js]: Excellent idea! I recommend renaming this component and putting it somewhere else so you can use it throughout your whole app

File: src/pages/Add.js [hunk 4]
```diff
@@ -296,14 +314,13 @@ export default function Add() {
...
+              <i className="fas fa-euro-sign"></i>
               <i className="fas fa-euro-sign"></i>
             </label>
           </ChoicesItem>
         </MultipleChoicesBox>
-        <Link to="/thanks">
-          <Button type="submit">⇢ Ab geht's!</Button>
-        </Link>
+
+        <Button disabled={isLoading}>⇢ Ab geht's!</Button>
```
Comment (marwinburesch) [inline src/pages/Add.js]: nice

File: src/pages/Add.js (hunk 3 above)
Comment (nadinho) [reply src/pages/Add.js]: Yeah, good idea :)

**Pull request merged**